src/kernel/src/ee/llvm/mod.rs

```diff
@@ -11,10 +11,12 @@ mod codegen;
 
 /// An implementation of [`ExecutionEngine`] using JIT powered by LLVM IR.
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Better to remove `llvm` field instead of applying this one because of the same previous reason.